src/powershell/Public/Initialize-FinOpsHubDeployment.ps1

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    Initialize a FinOps hub deployment in order to enable resource group owners to deployment hubs via the portal.
+
+    .PARAMETER WhatIf
+    Optional. Shows what would happen if the command runs without actually running it.
+
+    .EXAMPLE
+    Initialize-FinOpsHubDeployment -WhatIf
+
+    Shows what would happen if the command runs without actually running it.
+
+    .DESCRIPTION
+    The Initialize-FinOpsHubDeployment command performs any initialization tasks required for a resource group contributor to be able to deploy a FinOps hub instance in Azure, like registering resource providers. To view the full list of tasks performed, run the command with the -WhatIf option.
+
+    .LINK
+    https://aka.ms/ftk/Initialize-FinOpsHubDeployment
+
+#>
+
+
+function Initialize-FinOpsHubDeployment {
+    [CmdletBinding(SupportsShouldProcess)]
+    param()
+
+    if ($PSCmdlet.ShouldProcess('Required resource providers', 'Register')) {
+            # Register required resource providers
+            Write-Verbose "Registering required resource providers for FinOps Hub deployment."
+            Register-FinOpsHubProviders -WhatIf:$WhatIfPreference -Verbose:$VerbosePreference
```

Comment:
```suggestion
            Register-FinOpsHubProviders
```

Don't need to pass whatif through since it won't be called if it's actually used. Also don't need to pass verbose through